README.md

```diff
@@ -167,13 +167,13 @@ project's clang-cl compiler, like below:
     path/to/configure CC="path/to/msvcc.sh -clang-cl" CXX="path/to/msvcc.sh -clang-cl" LD=link CPP="clang-cl -EP"
 
 When building with MSVC under a MingW environment, you may need to
-remove the line in configure that sets 'fix_srcfile_path' to a 'cygpath'
-command.  ('cygpath' is not present in MingW, and is not required when
+remove the line in configure that sets `fix_srcfile_path` to a `cygpath`
+command.  (`cygpath` is not present in MingW, and is not required when
 using MingW-style paths.)
 
-To build static library for ARM64 with MSVC using visual studio solution, msvc_build folder have
-   aarch64/Ffi_staticLib.sln
-   required header files in aarch64/aarch64_include/
+To build static library for ARM64 with MSVC using visual studio solution, *msvc_build* folder have
```

Comment:
Now that I read this sentence again, it seems to be missing words.  Should it be "must have"?

Reply:
There was another rewrite of the readme that somebody submitted a few months ago.  I never merged it.  Did they rewrite this section?